.github/ISSUE_TEMPLATE/04-binding-a-java-library.yml

```diff
@@ -12,20 +12,21 @@ body:
   - type: dropdown
     id: android-type
     attributes:
-      label: Android application type
-      description: In what type(s) of Android application(s) do you see this issue?
+      label: Android framework version
+      description: In what target framework(s) of Android application(s) do you see this issue?
       multiple: true
       options:
-      - Classic Xamarin.Android (MonoAndroid13.0, etc.)
-      - .NET Android (net7.0-android, net8.0-android, etc.)
+      - net8.0-android
+      - net9.0-android
+      - Other
     validations:
       required: true
   - type: input
     id: platform-versions
     attributes:
       label: Affected platform version
       description: Please provide the version number of the platform you see this issue on.
-      placeholder: E.g. VS 2022 17.8.0, VSMac 17.6.7, .NET 8.0.100, etc.
+      placeholder: E.g. VS 2022 17.9.0, .NET 8.0.100, etc.
```

Comment:
Should we ask for the VS Code version? Maybe just the version of the MAUI extension?

Reply:
I believe the intent here is to help us determine what version of our code they are using.  Since VSCode doesn't ship our bits, I don't know if its version helps any.  For VSCode users I suspect we would rather they provide the .NET version.